tests/integration_tests/functional/test_cpu_features_aarch64.py

```diff
@@ -63,6 +64,37 @@ def get_cpu_template_dir(cpu_template):
     return cpu_template if cpu_template else "none"
 
 
+@pytest.mark.skipif(
+    PLATFORM != "aarch64",
+    reason="This is aarch64 specific test.",
+)
```

Comment:
We could make this test more generic and support x86_64. Maybe it belongs in a generic `test_cpu_features.py` (and move the current `test_cpu_features.py` to `test_cpu_features_x86.py`)

Reply:
I would prefer to keep our current separation where `test_cpu_feautres` is for x86 and `test_cpu_features_aarch64` is for aarch64. No reason for creating some new file for a singe test.
I don't mind renaming `test_cpu_features` to `test_cpu_features_x86` though.

Reply:
On the other hand, we should look for any opportunities to consolidate and unify those tests. I think this test could be the first step in that direction.

Reply:
These are 2 different architectures, we don't need to unify tests for them.

Reply:
I disagree. If the test is the same, we don't need to separate tests just because of different architecture. It's less effort to maintain and fix one test than two. Some CPU tests may be very specific and apply to only one architecture, but that's not a good argument to duplicate some tests.

Reply:
> It's less effort to maintain and fix one test than two.

Reply:
To me number of files is not that important. As I see it, it's better to have 1 new file with only one test that handles all architectures, than 2 tests in existing files.

But anyway, I do have some unrelated changes in these files so I can do this follow up work then.